pkg/daemon/ceph/client/crash_test.go

```diff
@@ -39,7 +40,7 @@ var (
 func TestCephCrash(t *testing.T) {
 	executor := &exectest.MockExecutor{}
 	context := &clusterd.Context{Executor: executor}
-	executor.MockExecuteCommandWithOutput = func(command string, args ...string) (string, error) {
+	executor.MockExecuteCommandWithOutput = func(timeout time.Duration, command string, args ...string) (string, error) {
```

Comment:
Instead of adding `timeout` to the function, let's call `ExecuteCommandWithTimeout()`, which already has the timeout as an arg.

Reply:
?